dev-libs/libstrophe/libstrophe-0.10.0.ebuild

```diff
@@ -40,7 +41,8 @@ src_compile() {
 src_install() {
 	default
 	use doc && dodoc -r examples
-	find "${D}" -name '*.la' -o -name '*.a' -delete || die
+	find "${D}" -name '*.la' -delete || die
+	find "${D}" -name '*.a' -delete || die
```

Comment:
You sure the previous find wasn't working? Hm.

Reply:
I was confused, too, but the previous find left the la files in place, which portage of course complained about

Reply:
I usually place the ORed parts of `find` conditions in braces. And explicitly specify `-type f`. So maybe

```
find "${D}" -type f \( -name '*.la' -o -name '*.a' \) -delete || die
```

would do the trick?

Reply:
Even though, I believe `-delete` may not need `-type f` to avoid running into applying the `find` action to directories: the man page of find gives the impression that `-delete` only operates on files anyway.

Reply:
Yes, it did. So probably `-delete` only applied to the right-hand side of `-o` before?

Reply:
Great. Yes, I assume that previously `-delete` only applied to the right hand side. Meanwhile I also confirmed that it is sensible to add `-type f`, as otherwise `-delete` will also try to delete matching directories and return with an error if the directory is not empty. Not that I believe this to be frequently the case, but using `-type f` makes the whole thing more robust.

Reply:
Updated to commit 26f8c21. This one now has the find call which you've proposed.